### ISSUE TITLE
Large model architecture changed?

### ISSUE THREAD

**Issue by picosankaricpp** [roboflow/rf-detr#830]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

I trained some RFDETR large models towards the end of January. They performed well and had best_ema.pths that were about 1.5 GB. I just tried training a new model RFDETR Large model and got a much smaller best_ema.pth (~0.4 GB) and substantially worse performance. I'm not sure what RFDETR version I had installed when training past models, but it may have been updated. Have other people experienced this? Am I missing something?

### Environment

rfdetr: 1.4.0.post0

### Minimal Reproducible Example
```py
model = RFDETRLarge()
        model.train(
            dataset_dir=dataset_path,
            epochs=epochs,
            batch_size=batch_size,
            grad_accum_steps=grad_accum_steps,
            lr=lr,
            output_dir=output_dir,
            early_stopping = True,
            tensorboard=True,
            early_stopping_patience=7,
            num_workers = 0
        ) #fill out with your own dataset_path and output_dir
```

### Are you willing to submit a PR?

- [ ] Yes, I'd like to help by submitting a PR!

**Comment by picosankaricpp** [comment]
I used this script to check parameters: 
```py
def model_pth_comparison(model_pth_1, model_pth_2):
    import torch

    def inspect(path):
        ckpt = torch.load(path, map_location="cpu", weights_only=False)
        state = ckpt["model"] if isinstance(ckpt, dict) and "model" in ckpt else ckpt

        params = sum(v.numel() for v in state.values())

        #print("file:", path)
        print("keys:", list(ckpt.keys()) if isinstance(ckpt, dict) else "state_dict only")
        print("parameters:", params)

        for k, v in state.items():
            print("dtype:", v.dtype)
            break

    inspect(model_pth_1)
    inspect(model_pth_2)
```
My old .pth has 135152374 parameters while my new .pth has 33609398. When did this change?

**Comment by picosankaricpp** [comment]
Just to make sure, I retrained a current 'Large' model on the exact same dataset as my old large model. The results are different (different metrics at different thresholds) and a little worse. I think that what was originally RFDETR Large is now XL or 2XL

**Comment by Borda** [comment]
@picosankaricpp — great detective work tracking down the parameter count change!
You're right that the Large architecture was intentionally redesigned in v1.4.0 using NAS-optimized configurations for a better accuracy-speed tradeoff at that
model tier.

If you need the parameter budget closer to the old Large (~135M), the XL model (`pip install rfdetr[plus]`) is the closest equivalent. v1.6.0 also introduced composable Lightning training that makes switching between model sizes seamless.

Appreciate the thoroughness of your analysis — closing as clarified, but absolutely reopen if you need help migrating workflows to the new architecture!
